parking_permits/customer_permit.py

```diff
@@ -488,7 +491,7 @@ def _calculate_prices(self, permit, product_with_qty):
         quantity = product_with_qty[1]
 
         base_price = unit_price = product.unit_price
-        discount_price = base_price - (product.low_emission_discount * unit_price)
```

Comment:
Tidy up: not a functional change, but `base_price` is clearer.

Reply:
Good!